solutions/devicesimulation-nohub/single-vm/setup-internal.sh

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env bash
```

Comment:
We are losing history by adding setup-internal.sh as a new file, we should rename setup.sh file to setup-internal.sh and add a new file setup.sh. Not sure if this is done already.

Reply:
We tried it but Git does not seem to respect that. Both Avani and Devis tried that.

Reply:
That seems weird. I'll approve since both have tried it and i don't really have cycles to look into it :)